server/pbench/bin/pbench-satellite-cleanup

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+import sys
+import os
+import shutil
+for line in sys.stdin:
+    tarball = line.split(": ")[1]
+    try:
+        src = "TO-SYNC/%s" % tarball
+        des = "SYNCED"
+        shutil.move(src, des)
+    except Exception as e:
+        print (e)
+        continue
+    try:
+        os.remove('%s' % tarball)
+    except Exception as e:
+        print (e)
+    try:
+        os.remove('%s.md5' % tarball)
+    except Exception as e:
+        print (e)
```

Comment:
If we get an error here, will this get cleaned up by another process later (from the department of the redundancy department)?